cypress/integration/smoke/RESTInteractionTest.ts

```diff
@@ -109,7 +109,7 @@ onlyOn(isEnvironmentType(EnvType.Dev), () => {
         .within(() => {
           cy.get("td:first").should("have.text", bridgeName);
           cy.get("td:nth-child(3)").then(($state) => {
-            cy.wrap($state, { timeout: 60000 }).should("have.text", "Ready");
+            cy.wrap($state, { timeout: 240000 }).should("have.text", "Ready");
```

Comment:
This timeout is similar like in https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/src/test/resources/features/bridge.feature

Reply:
Should we also increase timeouts during deletion checks? See [comment on the other PR](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/226#issuecomment-1281977401)